internal/web/templates/layout/base.html

```diff
@@ -115,9 +116,13 @@ <h1 class="navbar-brand text-white mx-1 my-0">Glasskube</h1>
             {{ end }}
           </div>
           <div class="col-4 text-end">
-            <a class="nav-link" href="https://glasskube.cloud/signup.html?id={{ .CloudId }}" target="_blank"
-              ><span class="bi bi-box-arrow-up-right me-1"></span>Glasskube Cloud</a
-            >
+            <a class="nav-link" href="https://glasskube.cloud/signup.html?id={{ .CloudId }}" target="_blank">
+              <span class="bi bi-box-arrow-up-right me-1"></span>Glasskube Cloud
+            </a>
+            <br />
```

Comment:
This leads to an unnecessary gap (see screenshot)

```suggestion
```

![image](https://github.com/glasskube/glasskube/assets/16959694/65df91e4-8b6f-4990-a153-78ca00611123)